ptbcontrib/extbotcommand/extbotcommand.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+#
+# A library containing community-based extension for the python-telegram-bot library
+# Copyright (C) 2020-2021
+# The ptbcontrib developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""
+This module contains a subclass for the BotCommand class
+that allows for a longer description (e. g. for /help purposes).
+"""
+from telegram import BotCommand
+
+
+class ExtBotCommand(BotCommand):
```

Comment:
Maybe rather `LongBotCommand`? `Ext` sounds a bit too generic IMHO, as this class provides a very specific extension. One could also just name it `BotCommand` again …

Reply:
Done, I feel a bit uncomfortable naming a subclass the same thing as its parent class just yet